test/server/proxy/regression-test.js

```diff
@@ -265,6 +265,8 @@ describe('Regression', () => {
     });
 
     it('Should raise error on request timeout (T224541)', done => {
+        session.options.requestTimeout = {};
```

Comment:
Does it still necessary?

Reply:
Yes, I deleted default timeouts for the session.